.github/actions/centos-build/entrypoint.sh

```diff
@@ -19,7 +19,7 @@ git clone --depth 1 -b $1 https://github.com/ImageMagick/ImageMagick.git
+    sed -i '/AutoReq: no/a Requires: /sbin/ldconfig /sbin/ldconfig config(ImageMagick-libs) = 7.0.10-59 libHalf.so.6()(64bit) libICE.so.6()(64bit) libIex.so.6()(64bit) libIexMath.so.6()(64bit) libIlmImf.so.7()(64bit) libIlmThread.so.6()(64bit) libImath.so.6()(64bit) libSM.so.6()(64bit) libX11.so.6()(64bit) libXext.so.6()(64bit) libXt.so.6()(64bit) libbz2.so.1()(64bit) libc.so.6()(64bit) libc.so.6(GLIBC_2.11)(64bit) libc.so.6(GLIBC_2.14)(64bit) libc.so.6(GLIBC_2.17)(64bit) libc.so.6(GLIBC_2.2.5)(64bit) libc.so.6(GLIBC_2.3)(64bit) libc.so.6(GLIBC_2.3.4)(64bit) libc.so.6(GLIBC_2.4)(64bit) libcairo.so.2()(64bit) libfftw3.so.3()(64bit) libfontconfig.so.1()(64bit) libfreetype.so.6()(64bit) libgcc_s.so.1()(64bit) libgcc_s.so.1(GCC_3.3.1)(64bit) libgdk_pixbuf-2.0.so.0()(64bit) libgio-2.0.so.0()(64bit) libglib-2.0.so.0()(64bit) libgobject-2.0.so.0()(64bit) libgomp.so.1()(64bit) libgomp.so.1(GOMP_1.0)(64bit) libgomp.so.1(OMP_1.0)(64bit) libgomp.so.1(OMP_3.0)(64bit) libjbig.so.2.0()(64bit) libjpeg.so.62()(64bit) libjpeg.so.62(LIBJPEG_6.2)(64bit) liblcms2.so.2()(64bit) libltdl.so.7()(64bit) liblzma.so.5()(64bit) liblzma.so.5(XZ_5.0)(64bit) libm.so.6()(64bit) libm.so.6(GLIBC_2.2.5)(64bit) libopenjp2.so.7()(64bit) libpango-1.0.so.0()(64bit) libpangocairo-1.0.so.0()(64bit) libpng15.so.15()(64bit) libpng15.so.15(PNG15_0)(64bit) libpthread.so.0()(64bit) libpthread.so.0(GLIBC_2.2.5)(64bit) librsvg-2.so.2()(64bit) libtiff.so.5()(64bit) libtiff.so.5(LIBTIFF_4.0)(64bit) libwebp.so.4()(64bit) libwmflite-0.2.so.7()(64bit) libxml2.so.2()(64bit) libxml2.so.2(LIBXML2_2.4.30)(64bit) libxml2.so.2(LIBXML2_2.6.0)(64bit) libz.so.1()(64bit) rtld(GNU_HASH)' ImageMagick.spec.in && \
```

Comment:
why this change?

Reply:
This are the rpm libs which needs to be updated as per the imagemagick version.